src/Hazelcast.Net/DistributedObjects/Impl/HMap.Setting.cs

```diff
@@ -392,9 +392,15 @@ Task SetTransientAsync(IData keyData, IData valueData, TimeSpan timeToLive, Time
 #endif
         }
 
-        public Task<bool> UpdateTimeToLive(TKey key, TimeSpan timeToLive)
+        public async Task<bool> UpdateTimeToLive(TKey key, TimeSpan timeToLive)
         {
-            throw new NotImplementedException();
+            var keyData = ToSafeData(key);
```

Comment:
At java the key and TimSpan are checked for null? will you add the checks?

Reply:
It's a struct cannot be null.